internal/earlyrand/earlyrand.go

```diff
@@ -0,0 +1,33 @@
+// Copyright 2018 - The Ignition authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package earlyrand
+
+import (
+	"fmt"
+	"io"
+	"os"
+)
+
+var urandomPath = "/dev/urandom"
```

Comment:
const?

Reply:
`var` can be directly overridden via linker flags, but that's probably not very useful here. I'll const-ify it.